docs/src/components/typography/font-weight.tsx

```diff
@@ -5,64 +5,70 @@ import * as styles from './font-weight.module.css'
 
 const weights = [
   {
-    label: 'BLACK',
+    label: 'Default (400)',
+    weight: 400,
+    varName: 'psTypeFontWeightDefault',
+    jsVarName: 'type.fontWeightDefault'
+  },
+  {
+    label: 'Strong (600)',
+    weight: 600,
+    varName: 'psTypeFontWeightStrong',
+    jsVarName: 'type.fontWeightStrong'
+  },
```

Comment:
Reading the scale is a bit unexpected, since it goes light -> heavy -> light.  Might be nice to put an hr between the default stuff and the linear scale (for weights and size)

<img width="653" alt="Screen Shot 2021-01-14 at 1 28 42 PM" src="https://user-images.githubusercontent.com/552044/104645818-d4b4af80-566c-11eb-99c2-91f044b80e46.png">

Reply:
RE: storybook These were both issues in master unrelated to the changes in this PR.

I changed the text from "orange" to "blue" to make it honest. I think that was a hold over from when orange was our primary action color. 

I'll add a card for fixing the mixed list story, because that's beyond my ability.